1.data_preprocessing/1.make_pseudobulks.ipynb

```diff
@@ -0,0 +1,927 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        stratify=adata.obs[[SAMPLE_ID_COL, STIM_COL, CELL_TYPE_COL]],
</pre><p>I think the ideal split would be to hold out an entire sample. BuDDIs typical usage is to hold out the "perturbed" samples, but we don't really have this in this experiment. How are you thinking about making the split? Or what is your hypothesis driving this type of split?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NDavidsonLab/BuDDI_HGSC_schildkraut_analysis/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>